pkg/notify/email.go

```diff
@@ -58,47 +60,53 @@ func NewEmailNotifier(logger log.Logger, val interface{}) Notifier {
 	}
 	notifier.Template = tmpl
 
+	if opts.NotificationTimeout.Email != nil {
```

Comment:
Options, NotificationTimeout, Email are all optional , so we need to check if they are nil or not